backend/src/main/dto/TaskDTOs.ts

```diff
@@ -5,7 +5,7 @@ export interface TaskCreationDTO {
   deadlineDate: Date;
   createdDate: Date;
   modifiedDate: Date;
-  projectId: number;
+  projectId?: number;
```

Comment:
Do we allow creating a task without assigning it to a project?

Reply:
Yeah from the acceptance criteria, but currently the database doesn't allow it, so Yup marks it as required.